cmd/namespace.go

```diff
@@ -118,17 +147,32 @@ To change the namespace, you will have to contact CircleCI customer support.
 
 	confirm := fmt.Sprintf("Are you sure you wish to create the namespace: `%s`", namespaceName)
 	if opts.noPrompt || opts.tty.askUserToConfirm(confirm) {
-		_, err := api.CreateNamespace(opts.cl, namespaceName, opts.args[2], strings.ToUpper(opts.args[1]))
-
+		_, err = api.CreateNamespace(opts.cl, namespaceName, opts.args[2], strings.ToUpper(opts.args[1]))
 		if err != nil {
 			return err
 		}
 
 		fmt.Printf("Namespace `%s` created.\n", namespaceName)
 		fmt.Println("Please note that any orbs you publish in this namespace are open orbs and are world-readable.")
 	}
+	return err
+}
 
-	return nil
+func createNamespace(opts namespaceOptions) error {
+	var err error
+	namespaceName := opts.args[0]
+	//skip if no orgid provided
+	if opts.orgID != nil && strings.TrimSpace(*opts.orgID) != ""{
+		_, err = uuid.Parse(*opts.orgID)
+		if err == nil {
+			return createNamespaceWithOrgId(opts, namespaceName, *opts.orgID)
+		}
+
+	//skip if no vcs type and org name provided
+	} else if len(opts.args) > 1{
```

Comment:
should check for len == 3, otherwise `opts.args[2]` will throw if there's only 2 args